frontend/src/components/SampleQueries.less

```diff
@@ -22,5 +22,17 @@
         text-overflow: ellipsis;
         cursor: pointer;
         padding-right: 40px;
+
+        svg path {
```

Comment:
I see you repeat this rules multiple times. Do you think we can create function `icon-variant` to use here? And reuse it in `icon-btn-variant`?

Reply:
I did a small refactor in 1de485b955aea153f3ca200ec272c70e077f2234.